unabridged-works-volume-3/original/11-flesh-blood-mystery.adoc

```diff
@@ -930,15 +972,15 @@ sanctifying, yea, and justifying also), than any but those only that are born of
 and kept alive by him, yet ever knew, or can know.
 
 [.centered]
-=== POSTSCRIPT
+=== Postscript
 
 [.heading-continuation-blurb]
 Containing a few Words concerning the Doings and
 Sufferings of that despised People called Quakers,
 which are both misunderstood and misrepresented by many:
 with an Exhortation to true Christianity.
 
-FIRST, their doings are looked upon by many to be from a natural principle,
+First, their doings are looked upon by many to be from a natural principle,
```

Comment:
So I didn't make this into a numbered group because there was no "secondly" or anything thereafter that I could see. But it's early in the morning and I may have missed it! 😬

Reply:
I think that's fine 👍